pkgs/development/libraries/crypto++/default.nix

```
@@ -21,6 +21,16 @@ stdenv.mkDerivation rec {
     substituteInPlace GNUmakefile \
         --replace "AR = libtool" "AR = ar" \
         --replace "ARFLAGS = -static -o" "ARFLAGS = -cru"
+
+    # See https://github.com/weidai11/cryptopp/issues/1011
+    substituteInPlace GNUmakefile \
+      --replace "ZOPT = -O0" "ZOPT ="
+  '';
+
+  configurePhase = ''
+    runHook preConfigure
+    sh TestScripts/configure.sh
+    runHook postConfigure
   '';
```

```suggestion
  preConfigure = ''
    sh TestScripts/configure.sh
  '';
```

Can we do this instead?

---

Sure, amended.